src/config.cc

```diff
@@ -151,9 +151,6 @@ void Config::postUpdateValues() {
   if (uptane.director_server.empty()) uptane.director_server = tls.server + "/director";
 
   if (uptane.ostree_server.empty()) uptane.ostree_server = tls.server + "/treehub";
-
-  uptane.public_key_path = (tls.certificates_directory / uptane.public_key_path).string();
-  uptane.private_key_path = (tls.certificates_directory / uptane.private_key_path).string();
```

Comment:
In my opinion modification of config variables makes it a bit harder to trace the state of the system while reading the code. And all these pathes/file names are only used (or at least they should only be used) in fsstorage.cc and the code there is quite compact, so I don't think longer expressions is a big issue here.